css/properties/width.json

```
@@ -286,6 +286,54 @@
             }
           }
         },
+        "fit-content_function": {
+          "__compat": {
+            "description": "<code>fit-content()</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
```

Found a tracking bug:

```suggestion
                "version_added": false,
                "notes": "See <a href='https://bugzil.la/1312588'>bug 1312588</a>."
```